pkg/template/template.go

```diff
@@ -8,41 +8,39 @@ import (
 	"path/filepath"
 	"strings"
 	textTemplate "text/template"
-	"time"
-
-	"github.com/cyberark/conjur-oss-suite-release/pkg/changelog"
 )
 
-// SuiteComponent represents a suite component with all of its changelogs and
-// relevant pin data
-type SuiteComponent struct {
-	CertificationLevel string
-	Changelogs         []*changelog.VersionChangelog
-	ReleaseName        string
-	ReleaseDate        string
-	Repo               string
-	UpgradeURL         string
-	URL                string
+type TemplateInfo struct {
+	TemplateName   string
+	OutputFileName string
 }
 
-// ReleaseSuite stores all the data needed for generation of templates in the suite
-type ReleaseSuite struct {
-	Version          string
-	Date             time.Time
-	Components       []SuiteComponent
-	UnifiedChangelog string
+var templates = map[string]TemplateInfo{
+	"changelog": {
+		TemplateName:   "CHANGELOG_unified.md.tmpl",
+		OutputFileName: "CHANGELOG_%s.md",
+	},
+	"release": {
+		TemplateName:   "RELEASE_NOTES_unified.md.tmpl",
+		OutputFileName: "RELEASE_NOTES_%s.md",
+	},
+	"unreleased": {
+		TemplateName:   "UNRELEASED_CHANGES_unified.md.tmpl",
+		OutputFileName: "UNRELEASED.md",
+	},
```

Comment:
This structure is highly irrelevant to the "template" package itself which should just know how to create templates. I would probably move this into the cli package since the CLI is the one that cares where you want to output the file.

Reply:
Agreed - moved to cli package